test/srt/test_openai_server.py

```diff
@@ -125,9 +125,6 @@ def run_completion_stream(self, echo, logprobs, token_input):
 
             assert response.id
             assert response.created
-            assert response.usage.prompt_tokens > 0
```

Comment:
add assert somewhere or test `stream_options?

Reply:
add new assert in test in the latest commit